packages/typespec-python/test/generic_mock_api_tests/asynctests/test_typetest_property_additionalproperties_async.py

```diff
@@ -18,10 +18,10 @@ async def client():
     "og_name,val, model_name", [
         ("extends_unknown", {'name': 'ExtendsUnknownAdditionalProperties', 'prop1': 32, 'prop2': True, 'prop3': 'abc'}, models.ExtendsUnknownAdditionalProperties),
         ("extends_unknown_derived", {'name': 'ExtendsUnknownAdditionalProperties', 'index': 314, 'age': 2.71828, 'prop1': 32, 'prop2': True, 'prop3': 'abc'}, models.ExtendsUnknownAdditionalPropertiesDerived),
-        ("extends_unknown_discriminated", {'name': 'Derived', 'index': 314, 'age': 2.71828, 'prop1': 32, 'prop2': True, 'prop3': 'abc'}, models.ExtendsUnknownAdditionalPropertiesDiscriminatedDerived),
+        ("extends_unknown_discriminated", {'kind': 'derived', 'name': 'Derived', 'index': 314, 'age': 2.71828, 'prop1': 32, 'prop2': True, 'prop3': 'abc'}, models.ExtendsUnknownAdditionalPropertiesDiscriminatedDerived),
```

Comment:
Updating these tests to match with what I see in cadl-ranch: https://github.com/Azure/cadl-ranch/pull/479/files#diff-e96e1cba09706c8a0e00bde73f721986543b99e3a0db21b3bbb98559db31185a